src/components/DateRangePicker.jsx

```diff
@@ -469,8 +473,10 @@ class DateRangePicker extends React.PureComponent {
 
     const withAnyPortal = withPortal || withFullScreenPortal;
 
+    /* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Is this not doable in the same inline comment?

Reply:
oooh TIL https://stackoverflow.com/a/56714489